js/main.js

```diff
@@ -34,6 +41,92 @@ Site = {
   },
 };
 
+Site.Mailchimp = {
+  $submitButton: Site.$mailchimpForm.find('input[type="submit"]'),
+  $emailInput: Site.$mailchimpForm.find('input[type="email"]'),
+
+  init: function() {
+    var _this = this;
+      
+    _this.$submitButton.on('click', function (event) {
+      event.preventDefault();
+      _this.$submitButton.prop('disabled', true);
+
+      if (_this.validateEmail(_this.$emailInput.val())) { 
+        _this.mailchimpAjax(Site.$mailchimpForm); 
+      } else {
+        _this.printResponse('invalid-email');
+      }
+    });
+  },
+
+  validateEmail: function(email) {
+    var regex = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
+    var is_email = regex.test(email);
+
+    if (is_email) {
+      return true;
+    }
+    return false;
+  },
+
+  mailchimpAjax: function($form) {
+    var _this = this;
+
+    $.ajax({
+      type: $form.attr('method'),
+      url: $form.attr('action'),
+      data: $form.serialize(),
+      cache: false,
+      dataType: 'jsonp',
+      jsonp: 'c',
+      contentType: "application/json; charset=utf-8",
+      error: function(err) { 
+        _this.printResponse('server-error');
+      },
+      success: function(data) {
+        if (data.result != 'success') {
+          _this.printResponse('invalid-email');
+        } else {
+          _this.printResponse('success');
+        }
+      },
+    });
+  },
+
+  printResponse: function(code) {
+    var _this = this,
+      $responseElem = $('#mailchimp-response');
+
+    switch (code) {
+      case 'invalid-email':
+        if (currentLang == 'es') {
+          $responseElem.html('Tu correo está raro...inténtalo de nuevo');
```

Comment:
using localized script drawing from site options it would be possible to have these set in admin. But probably not worth the distraction rn so should be just check the copy with Brett?